fluentforms/core/src/main/java/com/_4point/aem/fluentforms/impl/assembler/AdobeAssemblerResultImpl.java

```diff
@@ -0,0 +1,26 @@
+package com._4point.aem.fluentforms.impl.assembler;
+
+import java.util.Map;
+
+import com._4point.aem.fluentforms.api.Document;
+import com._4point.aem.fluentforms.api.assembler.AssemblerResult;
+
+public class AdobeAssemblerResultImpl implements AssemblerResult {
+
+	
+	com.adobe.fd.assembler.client.AssemblerResult assemblerResult;
+	
+    private Map<String, Document> getDocuments;  
+    
+	@Override
+	public Map<String, Document> getDocuments() {
+		return getDocuments;
+	}
+	
+	public AdobeAssemblerResultImpl(com.adobe.fd.assembler.client.AssemblerResult assemblerResult) {
+		this.assemblerResult = assemblerResult;
+	}
+
```

Comment:
This class looks incomplete.  The constructor sets a private variable that is never read and the class provides an accessor for another private variable that is never initialized.